### ISSUE TITLE
Sanitize module defaults (Trac #258)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#258]
Go through offline software and ensure that all modules have reasonable and sane default values.

Migrated from https://code.icecube.wisc.edu/ticket/258
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T19:30:13", 
    "description": "Go through offline software and ensure that all modules have reasonable and sane default values.", 
    "reporter": "nega", 
    "cc": "blaufuss, olivas", 
    "resolution": "fixed", 
    "_ts": "1547235013440693", 
    "component": "combo core", 
    "summary": "Sanitize module defaults", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2011-05-11T20:28:15", 
    "milestone": "", 
    "owner": "blaufuss", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20110511T20:44:27: 
* nega changed owner from "blaufuss" to ""
* nega changed cc from "" to "blaufuss, olivas"

**Comment by icecube-trac** [comment]
Trac update at 20110713T00:38:22: blaufuss changed owner from "" to "blaufuss"

**Comment by icecube-trac** [comment]
Trac update at 20110713T00:46:56: 
* blaufuss changed resolution from "" to "fixed"
* blaufuss changed status from "new" to "closed"
* blaufuss commented:

> Only DOMcalibrator really has physics-impacting default params.  This really is a bigger issue for icerec.  
> 
> Two params stand out as possible to change, but given the usual usage of DOMcalibrator, best to leave with current
> defaults.  Other defaults seem updated to good values:
> 
> FADCTimeOffset -> 0.0
> CalibrateDataWithSLC -> False (since SLC/HLC are usually split before DOMcalibrator)
> 
> Item closed for Offline.

**Comment by icecube-trac** [comment]
Trac update at 20141123T03:37:57: 
* olivas commented:

> Milestone "Happy Time!" version of offline-software deleted


* olivas changed milestone from ""Happy Time!" version of offline-software" to ""

**Comment by icecube-trac** [comment]
Trac update at 20190111T19:03:44: nwhitehorn commented:

> In changeset 167959:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="167959"
> Fixes #258, thanks Claudio for the report.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T19:30:13: nwhitehorn commented:

> In changeset 168097:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168097"
> Fixes #258, thanks Claudio for the report.
> ```
